assets/src/dashboard/animations/utils/createKeyframeEffect.js

```diff
@@ -14,27 +14,26 @@
  * limitations under the License.
  */
 
-/**
- * Internal dependencies
- */
-import { ROTATION } from '../constants';
-
-export default function (type) {
-  const frames = {
-    [ROTATION.CLOCKWISE]: {
-      transform: ['rotateZ(0deg)', 'rotateZ(360deg)'],
-    },
-    [ROTATION.COUNTER_CLOCKWISE]: {
-      transform: ['rotateZ(0deg)', 'rotateZ(-360deg)'],
-    },
-    [ROTATION.PING_PONG]: {
-      transform: ['rotateZ(-45deg)', 'rotateZ(40deg)'],
-    },
-  };
+const AMPToWAAPIProps = {
+  iterations: {
+    infinity: Infinity,
+  },
+};
 
-  const keyframes = frames[type] || frames[ROTATION.CLOCKWISE];
+function createKeyframeEffect(target, keyframes, timings) {
+  // Translate AMP timings to Web API timings
+  const convertedTimings = Object.keys(timings).reduce(
+    (acc, property) => ({
+      ...acc,
+      [property]:
+        (AMPToWAAPIProps[property] &&
+          AMPToWAAPIProps[property][timings[property]]) ||
+        timings[property],
+    }),
+    {}
+  );
```

Comment:
I guess let's keep an eye on this. Not sure if we need a whole wrapper around the `KeyframeEffect` constructor if it's only to swap out `infinity` for `Infinity`. Cool with it for now assuming there's more irregularities down the pike between the two APIs.

However, if we get farther along and this is consistently the only thing we need to change between the two, might be worth making this a little less impactful

Reply:
I can dig it.  I only know of the `infinity` limitation.  Maybe once we finish creating all the base animations and rig up a template or two, we'll know better if any more props  need conversions or not (and then can simplify).